File: Program24.c
#include<stdio.h>
#include<stdbool.h>

bool ChkEven(int iNo)
{
    if((iNo % 2) == 0)
    {
        return true;
    }
    else
    {
        return false;
    }
}

int main()
{
    int iValue = 0;
    bool iRet;

    printf("Enter number\n");
    scanf("%d",&iValue);

     iRet = ChkEven(iValue);

     if(iRet == true)
     {
         printf("%d is even number",iValue);
     }
     else
     {
         printf("%d is odd number",iValue);
     }

    return 0;
}

File: Program24.java
import java.lang.*;
import java.util.*;

class Number
{
    public int iNo;

    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter number");
        this.iNo = sobj.nextInt();
    }

    public void Display()
    {
        System.out.println("Value is: "+this.iNo);
    }

    public boolean ChkEven()
    {
        if((iNo % 2) == 0)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

class Program24
{
    public static void main(String arg[])
    {
        boolean iRet ;

        Number nobj = new Number();

        nobj.Accept();
        nobj.Display();
        
        iRet = nobj.ChkEven();
        if(iRet == true)
        {
          System.out.print("Even number");
        }
        else
        {
          System.out.print("Odd number");
        }
    }
}